### DISCUSSION TITLE
TabControl styling issue

### DISCUSSION THREAD

**Discussion by OswyChow** [HandyOrg/HandyControl#657] [Q&A] [answered]
Hello guys, im pretty new to wpf development and development in general, im tackling my first application and i decided to use HandyControl because of its simplicity (tried some others and it was hell). I incurred into a problem with the tab control and was wondering if its normal, what happens is that after i set a style to the tabcontrol element for example `Style="{StaticResource TabControlCapsule}"` the other attributes that i specified, for example `IsScrollable="True"  ShowScrollButton="True"` get overridden and i cant make them work, i tried creating a style in the window resources as BasedOn but i cant seem to be able to override the style's ones. I noticed that if i dont explicitly set a style it all works fine except i dont have that neat selection line under the tabs when they get selected and i really want that.  
Any help will be greatly appreciated!

**Comment by ghost1372** [comment, marked as answer]
Hi, note that there are 2 types of tabcontrol 
- TabControl
example:

`<TabControl Style="{StaticResource TabControlCapsule}"/>
`
- Hc TabControl 
example:

`<hc:TabControl IsScrollable="True"/>
`

The important thing is that the styles only work in the **<TabControl** and Not **<hc:TabControl**
Also, properties like IsScrollable only work in **<hc:TabControl**

Please read the documentation

https://ghost1372.github.io/handycontrol/native_controls/tabControl/

https://ghost1372.github.io/handycontrol/extend_controls/tabControl/

**Reply by OswyChow** [reply]
Oh that makes sense! 
Is it possible to have that selection line under the tab element when selected like seen in the tabcontrol styles?

**Reply by ghost1372** [reply]
i think you need **TabControlInLine** style

**Reply by OswyChow** [reply]
> 
> 
> i think you need **TabControlInLine** style

Is it possible to have both the IsScrollable property (so using hc:TabControl) and the selection line that the TabControl's TabControlInLine style gives? Or is it something that just the native tab control can have?

**Reply by OswyChow** [reply]
> 
> 
> > i think you need **TabControlInLine** style
> 
> Is it possible to have both the IsScrollable property (so using hc:TabControl) and the selection line that the TabControl's TabControlInLine style gives? Or is it something that just the native tab control can have?

I just solved it by installing the custom version (HandyControls) and setting `IsUWPStyle = True`
Hope this will help someone who wants to achieve the same!
